src/main/java/org/moe/idea/builder/MOEModuleBuilder.java

```diff
@@ -110,7 +113,7 @@ public void setupRootModel(final ModifiableRootModel rootModel) throws Configura
         Project project = rootModel.getProject();
         Module module = rootModel.getModule();
 
-        myJdk = MOESdkType.getJDK();
+        myJdk = getLatestJDK();
```

Comment:
Maybe we could add a lower/upper java version bound? Something like `getJdkInBound(JavaSdkVersion min, JavaSdkVersion max)`
I think it could be a bit misleading to set e.g. a java 18 jdk, even if java 18 isn't supported by moe

Reply:
Yeah make sense. I was thinking if we should add a JDK selector on the new project wizard, similar to the screen when you creating a normal Gradle project in Intellij.

Reply:
multi-os-engine/multi-os-engine#180 created for considering adding the JDK selector. Don't worry about this for now.